curriculum/challenges/english/25-front-end-development/lab-leap-year-calculator/66c06fad3475cd92421b9ac2.md

```diff
@@ -74,7 +74,7 @@ assert.match(__helpers.removeJSComments(code), /isLeapYear\(\s*year\s*\)/);
 You should store the result of calling the `isLeapYear` function in a variable named `result`.
 
 ```js
-assert.match(__helpers.removeJSComments(code), /const\s+result\s*=\s*isLeapYear\(\s*year\s*\)/);
+assert.match(__helpers.removeJSComments(code), /(var|let|const)\s+result\s*=\s*isLeapYear\(\s*year\s*\)/);
```

Comment:
```suggestion
assert.match(__helpers.removeJSComments(code), /(?:var|let|const)\s+result\s*=\s*isLeapYear\(\s*year\s*\)/);
```
really small nit, but this can be a non-capturing group